includes/core/server/request/class-paginated.php

```diff
@@ -38,7 +33,7 @@ class Paginated extends Request {
 	 * @throws Invalid_Request_Parameter_Exception
 	 */
 	public function get_api(): string {
-		return $this->uri; // Will be changed on every request by overriding this method or use set_uri method.
+		return ''; // Will be changed on every request that extends this class.
```

Comment:
Abstract function override is only allowed since PHP version 7.2. Currently, the minimum supported version of WooCommerce payment plugin is 7.0, so I didn't want to introduce a possible break change. We should bump minimum PHP supported version to 7.2 or leave it like this.

Reply:
Can't we just remove it from the `Paginated` class though? I don't think it needs to be overwritten.

Reply:
![image](https://user-images.githubusercontent.com/15913573/208458093-e41ef2b8-b9fe-4f8a-b921-f204bb3155b8.png)
PHPStorm cries about missing method in class, so I added it 😄 and I think that all child classes need to fill "body" of all abstract methods that are defined in parent abstract class (maybe I am wrong on this one, so I am not 100% certain)